ITEvents/Screens/Search tab/SearchViewController.swift

```diff
@@ -12,10 +12,18 @@ class SearchViewController: UIViewController, ISearchView, ITabBarItemSelectable
     private var layout: DisplaySwitchLayout!
     private var layoutState: LayoutState!
     
+    private var fetchEventsDebounced: ((String, Bool) -> Void)!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         registerNibs()
         presenter.setup()
+        
+        fetchEventsDebounced = debounce(
```

Comment:
let's move this debouncing functionality to the Presenter.
As it is definitely not a view responsibility.
So in the view on every user pressed key,  or search button click we simply call:
```swift
presenter.searchBy(text: searchText, tags: [])
```